Library/Homebrew/formula_installer.rb

```diff
@@ -1076,7 +1076,7 @@ def post_install
       #{HOMEBREW_LIBRARY_PATH}/postinstall.rb
     ]
 
-    args << if formula.local_bottle_path.present?
+    args << if formula.local_bottle_path.present? && !formula.path.exist?
       formula.prefix/".brew/#{formula.name}.rb"
```

Comment:
```suggestion
    args << if !formula.path.exist? && (prefix_formula = formula.prefix/".brew/#{formula.name}.rb").exist?
      prefix_formula
```

Reply:
Thanks Mike. I applied your suggestion as-is, but wouldn't the formula always exist within the prefix? Do correct me if I'm wrong.

If it doesn't exist (but the formula doesn't exist in the Tap either), using `formula.path` wouldn't work.

Reply:
> I applied your suggestion as-is, but wouldn't the formula always exist within the prefix?

It should. It won't if I delete it 😂.

> If it doesn't exist (but the formula doesn't exist in the Tap either), using `formula.path` wouldn't work

Yup. Could always handle that case specifically and fail with a better message if desired. I've never seen it, though.